axelor-account/src/main/java/com/axelor/apps/account/service/PartnerAccountService.java

```diff
@@ -31,4 +41,13 @@ public String getDefaultSpecificTaxNote(Partner partner) {
 
     return fiscalPosition.getCustomerSpecificNoteText();
   }
+
+  @Transactional
```

Comment:
We can't update all partner MoveLines like that. We can have serveral records. Period and JPA cache is not managed, we can't update some old move lines.